package.json

```diff
@@ -57,7 +57,8 @@
     "tmp": "0.0.28",
     "unzip": "0.1.11",
     "when": "3.7.2",
-    "zip-dir": "1.0.0"
+    "zip-dir": "1.0.0",
+    "jszip": "^2.4.0"
```

Comment:
oh I just noticed the caret. I think we want to pin all dependencies to their exact version, like `"jszip": "2.4.0"` because otherwise someone cloning the repo for the first time may encounter a bug from using a newer version than what the existing developers were using. I know theoretically the caret should only pick up bug fixes but that can't be fully trusted :)